sos/plugins/npm.py

```diff
@@ -0,0 +1,56 @@
+# Copyright (C) 2014 Red Hat, Inc., Tomas Tomecek <ttomecek@redhat>
+
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU General Public License for more details.
+
+# You should have received a copy of the GNU General Public License
+# along with this program; if not, write to the Free Software
+# Foundation, Inc., 675 Mass Ave, Cambridge, MA 02139, USA.
+
+from sos.plugins import Plugin, RedHatPlugin, DebianPlugin, UbuntuPlugin, \
+    SuSEPlugin
+
+
+class Npm(Plugin, RedHatPlugin, DebianPlugin, UbuntuPlugin, SuSEPlugin):
+    """
+    Get info about available npm modules
+    """
+
+    requires_root = False
+    plugin_name = 'npm'
+    profiles = ('system',)
+
+    # in Fedora, Debian, Ubuntu and Suse the package is called npm
+    packages = ('npm',)
+    files = ('/opt/rh/rh-nodejs4/root/etc/npmrc', )
+    files = ('/etc/npmrc', )
```

Comment:
The truth is I committed that by accident. I'm sorry about that. I'm glad my work day is almost over :)

Those config files in `/etc` are not installed with the package, they are `%ghost` files. So in the end, there's no easy way to bind the plugin to a file.